### PR TITLE
CP host details enhancements 

### PR DESCRIPTION
**Pull request by amirfefer**
https://projects.theforeman.org/issues/33499

### CODE REVIEW

**Comment by ekohl** [comment]
Test failures look relevant.

File: webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    return sprintf(
+      __(
+        'Are you sure you want to delete host %s ? This action is irreversible'
+      ),
+      hostName
+    );
+  };
+
+  if (window.confirm(warningMessage())) {
```
Comment (amirfefer) [inline webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:40]: confirmation modal service merged after 3.0 branching, using the native confirm instead.

**Comment by tbrisker** [comment]
I think Foreman tests are failing due to the capybara pin, can you please CP https://github.com/theforeman/foreman/commit/8e45f67cad79fcc5ce1fe726d03ce4facc76d0ac as well?

**Comment by amirfefer** [comment]
@tbrisker, thanks!  updated.

**Review by tbrisker — APPROVED** [review]
Thanks @amirfefer ! test failure unrelated. leaving to @ekohl to merge once he's ready with the pr processor improvements.

**Pull request merged**